### DISCUSSION TITLE
Connection error

### DISCUSSION THREAD

**Discussion by PrathameshSolanke** [lllyasviel/Fooocus#2982] [Q&A]
I have downloaded the fooocus zip file , extracted it then I have run the realistic file after the fooocus opens in browser ever time I get connection error I also have latest drivers watched bunch of YouTube videos but nothing helps, please can anyone help me.Thank you

**Comment by ILikeToasters** [comment]
Your first place to look is in the CMD window and see if anything there. If you include that here it will possibly help solve the issue.

**Reply by PrathameshSolanke** [reply]
No errors in cmd window ,the only error I get is that connection error and whenever I prompt something I don't see the image generation process. If there is any other method please help me

**Comment by Rogala** [comment]
At the first start the program downloads additional models and lors for the preset you have chosen, check that you do not have vpn enabled and firewall does not block access to the Internet.

[Set the swap file to more than 40 GB if you have Windows](https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md#system-swap)

**Reply by PrathameshSolanke** [reply]
Do you have a lastest updated fooocus zip file that I can download and start using it.

**Reply by Rogala** [reply]
[The latest version is here, but it doesn't have all the models and lore in it, it's a very large size everything is uploaded later and according to your requests. In the original archive only the framework of the program in any case, you will have to download.](https://github.com/lllyasviel/Fooocus/releases/tag/2.3.1)

**Comment by octavioismael-fer** [comment]
Hello, I am encountering the same problem.

I am currently using Fooocus in a Google Colab notebook, practicing with different models and prompts, but a few days ago I received the same error.

What it does is cut off the script execution and the program terminates.
Sometimes it happens at stage 29/30, or other times it can't generate anything at all.

It always ends with the following: "^C"

If there is something I need to change or a new version to download, please let me know.

Regards
